doc/lightning-getroute.7.txt

```diff
@@ -21,11 +21,10 @@ There are two considerations for how good a route is: how low the
 fees are, and how long your payment will get stuck if a node goes down
 during the process.  The 'riskfactor' floating-point field controls
 this tradeoff; it is the annual cost of your funds being stuck (as a
-percentage), multiplied by the percentage chance of each node failing.
+percentage).
 
-For example, if you thought there was a 1% chance that a node would
-fail, and it would cost you 20% per annum if that happened,
-'riskfactor' would be 20.
+For example, if you thought the inconvenience of having funds stuck was
+worth 20% per annum interest, 'riskfactor' would be 20.
```

Comment:
Restating this to confirm I'm understanding: `riskfactor` is a way of dynamically valuing shorter routes, by accounting for the potential loss of liquidity in the case that a payment gets stuck along the route. i.e. the more hops in a route (and payment value) the higher the `riskfactor`. In some sense, this makes higher fee yet shorter routes more attractive as the possibility for getting stuck is lowered due to less chances of a link failure en route.